src/components/Toast/Toast.style.ts

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import checkLine from "@/assets/icons/check-line.svg";
+
+export const ToastContainer = styled.div`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+
+  gap: ${({ theme }) => theme.gap["4xs"]};
+  padding: ${({ theme }) => `${theme.gap["xs"]} ${theme.gap["2xl"]}`};
+
+  box-shadow: ${({ theme }) => theme.shadow.overlay};
+  border-radius: ${({ theme }) => theme.radius["sm"]};
+
+  opacity: ${({ theme }) => theme.opacity["visible"]};
+  background: ${({ theme }) => theme.colors?.light["object-hero"]};
+`;
+
+export const ToastLabelText = styled.span`
+  text-align: center;
+
+  ${({ theme }) => theme.typography?.label.bold["sm"]}
+  color: ${({ theme }) => theme.colors?.light["object-inv-hero"]};
+`;
+
+export const CheckLineImg = styled(checkLine)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  width: 1.25rem;
+  height: 1.25rem;
+
+  gap: ${({ theme }) => theme.gap["none"]};
+  padding: ${({ theme }) => theme.gap["none"]};
```

Comment:
저는 이런거 `lint` 오류 뜨던데.. 혹시 안떴나여??

```suggestion
  gap: ${({ theme }) => theme.gap.none};
  padding: ${({ theme }) => theme.gap.none};
```
이렇게 쓰라구..

그리구 다른 곳에서는 옵셔널 체이닝을 쓰셨는데 여기선 안쓰신 이유가 있으신가요? 안써도 동작하면 상관 없을 것 같긴 하지만.. ㅎㅎ

Reply:
정말 단순히 안 써도 동작했기 때문입니다...... 🤔 
`lint` 오류가 안 뜨던데 제 환경에서 혹시 `lint`가 작동 안 하고 있는 걸까요?

Reply:
그런 것 같아요..! 이부분은 제가 좀더 연구해볼게용! 
일단은 `npm run lint` 돌리시면 error나 warning 뜰거에여!